### PR TITLE
Fix `bootstrap_image` in mock-core-configs templates

### PR DESCRIPTION
**Pull request by gotmax23**
- core-configs: Update almalinux-8.tpl bootstrap_image
- core-configs: Use fully qualified bootstrap_image name

### CODE REVIEW

**Comment by xsuchy** [comment]
I do not see anything relevant on https://wiki.almalinux.org/containers/docker-images.html
Can you provide a link to an announcement? Or an official source?

**Review by Conan-Kudo — APPROVED** [review]

**Comment by Conan-Kudo** [comment]
> https://wiki.almalinux.org/containers/docker-images.html

Docker maps  `<name>` to `docker.io/library/<name>` automatically. This PR dereferences it.

File: mock-core-configs/etc/mock/templates/almalinux-8.tpl [hunk 1]
```diff
@@ -3,7 +3,7 @@ config_opts['dist'] = 'el8.alma'  # only useful for --resultdir variable subst
 config_opts['releasever'] = '8'
 config_opts['package_manager'] = 'dnf'
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
-config_opts['bootstrap_image'] = 'docker.io/almalinux/almalinux:8'
+config_opts['bootstrap_image'] = 'docker.io/library/almalinux:8'
```
Comment (Conan-Kudo) [inline mock-core-configs/etc/mock/templates/almalinux-8.tpl:6]: I would suggest using `quay.io/almalinux/almalinux:8` instead. [It's an official image](https://quay.io/repository/almalinux/almalinux) and updated more frequently.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Comment by gotmax23** [comment]
> I do not see anything relevant on https://wiki.almalinux.org/containers/docker-images.html Can you provide a link to an announcement? Or an official source?

Also, here[1] is the page that says that the previous image is deprecated.

[1]: https://hub.docker.com/r/almalinux/almalinux

File: mock-core-configs/etc/mock/templates/almalinux-8.tpl (hunk 1 above)
Comment (gotmax23) [reply mock-core-configs/etc/mock/templates/almalinux-8.tpl]: > I would suggest using `quay.io/almalinux/almalinux:8` instead. [It's an official image](https://quay.io/repository/almalinux/almalinux) and updated more frequently.

I just did. We could also switch to `quay.io/almalinux/8-base` which is UBI-compatible. Then, we would be in line with `rhel8.tpl`. What do you think?

Comment (Conan-Kudo) [reply mock-core-configs/etc/mock/templates/almalinux-8.tpl]: That would probably work too.

**Review by Conan-Kudo — APPROVED** [review]

**Comment by Conan-Kudo** [comment]
Thanks for the contributions!

**Pull request merged**